src/site/content/en/blog/responsiveness/index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: 'Feedback wanted: an experimental responsiveness metric'
+subhead: An update on our plans for measuring responsiveness on the web.
+description: An update on our plans for measuring responsiveness on the web.
+authors:
+  - hbsong
+date: 2021-11-03
+hero: image/eqprBhZUGfb8WYnumQ9ljAxRrA72/GNWyQZ5l1uy1b7s5ursN.jpeg
+alt: Water droplet rippling outward
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - performance
+  - web-vitals
+---
+
+Earlier this year, the Chrome Speed Metrics Team shared [some of the
+ideas](/better-responsiveness-metric/) we were considering for a
+new responsiveness metric. We want to design a metric that better captures the
```

Comment:
I'm assuming first person is fine in posts like this, given that it talks about what we (as in Google) are doing.

Reply:
yeah, the key thing is to be clear as to who is talking in that case, which this is.